docs/src/pages/index.page.tsx

```diff
@@ -133,69 +185,36 @@ const HomePage = ({ colorMode }) => {
             }}
             customSetup={{
               dependencies: {
-                '@aws-amplify/ui-react': '0.0.0-next-202192618169',
+                '@aws-amplify/ui-react': '0.0.0-next-5257a31-20211011212613',
```

Comment:
I didn't see CodeSandbox UI's supporting `next` tags.  Does it work here?
```suggestion
                '@aws-amplify/ui-react': 'next',
```